tools/ontoconvert

```diff
@@ -111,53 +182,129 @@ def main(argv: list = None):
+                ontologies.pop(0)
+            if ontologies:
+                for s, p, o in graph0.triples((ontologies[0], None, None)):
+                    graph2.add((iri, p, o))
+
+            # Serialise
+            if args.output_format:
+                fmt = FMAP.get(args.output_format, args.output_format)
+            else:
+                fmt = guess_format(args.output, fmap=FMAP)
+            graph2.namespace_manager.bind(
+                "emmo", rdflib.Namespace("http://emmo.info/emmo#")
             )
+            graph2.serialize(destination=args.output, format=fmt)
+
         else:
-            if not _validate_installed_version(
-                package="rdflib", min_version="6.0.0"
-            ) and (
-                output_format == FMAP.get("ttl", "")
-                or os.path.splitext(args.output)[1] == "ttl"
-            ):
-                from rdflib import (  # pylint: disable=import-outside-toplevel
-                    __version__ as __rdflib_version__,
-                )
 
-                warnings.warn(
-                    IncompatibleVersion(
-                        "To correctly convert to Turtle format, rdflib must be"
-                        " version 6.0.0 or greater, however, the detected "
-                        "rdflib version used by your Python interpreter is "
-                        f"{__rdflib_version__!r}. For more information see the"
-                        " 'Known issues' section of the README."
-                    )
-                )
+            onto.save(
+                args.output,
+                format=output_format,
+                dir=args.output_dir,
+                mkdir=True,
+                overwrite=args.overwrite,
+                recursive=args.recursive,
+                squash=args.squash,
+                write_catalog_file=bool(args.catalog_file),
+                append_catalog=args.append_catalog,
+                catalog_file=args.catalog_file,
+            )
+
+        # Bring back these lines when _apply_reasoning_results() works again
 
-            graph = Graph()
-            graph.parse(args.input, format=input_format)
-            graph.serialize(destination=args.output, format=output_format)
+        # if args.reasoner:
+        #     include_imported = not args.no_infer_imported
+        #     verbose = not args.quiet
+        #     onto.sync_reasoner(reasoner=args.reasoner,
+        #                        include_imported=include_imported,
+        #                        debug=verbose)
+        #
+        # onto.save(
+        #     args.output,
+        #     format=output_format,
+        #     dir=args.output_dir,
+        #     mkdir=True,
+        #     overwrite=args.overwrite,
+        #     recursive=args.recursive,
+        #     squash=args.squash,
+        #     write_catalog_file=bool(args.catalog_file),
+        #     append_catalog=args.append_catalog,
+        #     catalog_file=args.catalog_file,
+        # )
```

Comment:
Why having all this commented out?

Reply:
Because we want to bring back these lines again when the bug in sync_reasoner() has been fixed.